### DISCUSSION TITLE
Raspberry PI

### DISCUSSION THREAD

**Discussion by svitale623** [henrygd/beszel#104] [General]
can the hub and agent be installed on a RP 2?

**Comment by henrygd** [comment]
Yes, you should be able to use the linux arm binaries. And our [install script](https://github.com/henrygd/beszel/blob/main/supplemental/guides/systemd.md) should work as well.

The agent should consume less than 10MB RAM, but the hub may need 100 or more, so just be aware of that if you don't have much to spare.

Let me know if you have any trouble with it.

**Reply by svitale623** [reply]
Thanks, that seemed to work.  another question, how would i access the web interface outside the Pi itself.  i cant seem to access the web interface from within my network.  i confirmed both services are active and running.

**Reply by henrygd** [reply]
You should be able to use the ip or hostname of the pi with the port at the end. Do you have other applications on your pi that you are able to access from within your network?

My best guess is that you may have a firewall blocking access. See the [troubleshooting section of the readme](https://github.com/henrygd/beszel?tab=readme-ov-file#faq--troubleshooting).

Otherwise you can use SSH port forwarding / SSH tunneling, but that will require you to ssh into the pi first.

**Comment by svitale623** [comment]
Yes, currently running uptime kumA on the pi.  I can access this service by
hostname and IP

**Reply by henrygd** [reply]
What port are you running uptime kuma on?

And what is in the logs for the hub?

If you used the install script, run `systemctl status beszel` on the pi.

**Reply by svitale623** [reply]
uptimekuma is using the default port, 3001.  logs show ssh key pair generated successfully.  when i run the above command outputs - unit beszel.service could not be found.  if i run systemctl status beszel-hub.service, outputs running and active.

**Reply by henrygd** [reply]
I looked into this further and updated the install script for the hub to run on 0.0.0.0, which should provide access from outside.

I recommend downloading the hub install script again, running `-u` to uninstall, and installing fresh.

If you've already done some setup and don't want to start fresh, let me know and I'll give you the steps to update in place.

**Reply by svitale623** [reply]
great, thanks.  it working. :)
